src/main/startup.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  install,
+  uninstall
+}
+
+var startup = require('user-startup')
+
+function install () {
+  var appPath = process.execPath
+
+  startup.add('webtorrent-desktop', appPath)
```

Comment:
waiting for typicode/user-startup#7 to land,
apparently all params are required.